shared/oae/api/oae.api.util.js

```diff
@@ -1655,9 +1655,29 @@ define(['exports', 'require', 'jquery', 'underscore', 'oae.api.config', 'markdow
             if (!input) {
                 return '';
             } else {
-
+                var nonWord = /\W/;
                 // Escape asterisks instead of treating them as italics markers
-                input = input.replace(/\*/g, "\\*").replace(/\\\\/g, "\\");
+                input = input.replace(/\*/g, function(match, index) {
```

Comment:
I don't think they can, at least not nicely, as javascript doesn't allow for lookbehind in regex (so there isn't a way to check the preceding character). We would need to check every space and letter character to see if they are followed by an asterisk, then check the character following that asterisk - I think doing it this way is quicker.